app/models/register_search_result.rb

```diff
@@ -0,0 +1,12 @@
+class RegisterSearchResult < ActiveRecord::Base
+  self.primary_key = 'register_id'
+  scope :search, ->(search_term) { where(*['name ILIKE ? OR register_name ILIKE ? OR register_description ILIKE ?'].fill("%#{sanitize_sql_like(search_term)}%", 1..3)) }
```

Comment:
This should be approached with postgres full-text capabilities for performance sake. Either in this PR or in another one as long as the delay is not too much.

Reply:
I thought pg_search wouldn't see the JSONB given you are using a view. Ignore me then.

Reply:
good point. I have not tried using `pg_search` to query the view, but I'm not sure it buys us a lot even if it is supported. I will take a look.

Reply:
@arnau after some testing I have found for a table of this small size Postgres will elect to do a scan even when an index is available, so there is no benefit in adding one at this point. We could add a materialised view and re-index it if we think the extra overhead of refreshing it is worth the performance benefit.

Reply:
Ok thanks for looking into it 👍